packages/contracts/test/diamond/facets/UbiquityPoolFacet.t.sol

```diff
@@ -226,9 +226,9 @@ contract UbiquityPoolFacetTest is DiamondSetup {
         MockERC20 collateral = new MockERC20("collateral", "collateral", 18);
         collateral.mint(fourthAccount, 10 ether);
         // add collateral
-        IMetaPool metaPoolCollateral = IMetaPool(
-            address(new MockMetaPool(address(collateral), curve3CrvToken))
-        );
+        //IMetaPool metaPoolCollateral = IMetaPool(
```

Comment:
If we don't use it we can remove them

Reply:
pushed in https://github.com/ubiquity/ubiquity-dollar/pull/704/commits/0aad006a1c2d8a613d465e7e58c141285ff51b32